src/test/java/com/github/pjfanning/xlsx/StreamingSheetTest.java

```diff
@@ -363,7 +447,7 @@ public void testGetActiveCell() throws Exception {
     ) {
       Sheet sheet = workbook.getSheetAt(0);
       nextRow(sheet.rowIterator()); //need to force a read of first row before getActiveCell works
-      assertEquals(new CellAddress("A1"), sheet.getActiveCell());
+      assertEquals(new CellAddress("B2"), sheet.getActiveCell());
```

Comment:
I couldn't save the file to have the A1 cell selected - the tests read that as `null` in such case. So I selected the cell B2 and saved the file like that, hope that's ok